mapiproxy/servers/default/emsmdb/oxcfxics.c

```diff
@@ -1468,15 +1469,12 @@ static void oxcfxics_push_folderChange(struct emsmdbp_context *emsmdbp_ctx, stru
 			talloc_free(data_pointers);
 			talloc_free(retvals);
 			
-			/* TODO: check return code */
-			emsmdbp_object_open_folder(NULL, emsmdbp_ctx, folder_object, eid, &subfolder_object);
-			oxcfxics_push_folderChange(emsmdbp_ctx, synccontext, owner, topmost_folder_object, sync_data, subfolder_object);
-			talloc_free(subfolder_object);
-		}
-		/* else { */
-		/* 	DEBUG(5, ("no data returned for folder row %d\n", i)); */
-		/* 	abort(); */
-		/* } */
+			retval = emsmdbp_object_open_folder(NULL, emsmdbp_ctx, folder_object, eid, &subfolder_object);
+			if (retval == MAPISTORE_SUCCESS) {
+				oxcfxics_push_folderChange(emsmdbp_ctx, synccontext, owner, topmost_folder_object, sync_data, subfolder_object);
+				talloc_free(subfolder_object);
+			}
```

Comment:
I'm not questioning the usefulness of the debug message, I'm asking for help on defining a relevant one :wink: